sdk/eventhub/Microsoft.Azure.EventHubs.Processor/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 4.2.1 (Unreleased)
```

Comment:
Not sure if this is a relevant point, but the `Microsoft.Azure.EventHubs` series of projects are the track one client, which is owned by the service team.   I don't recall whether we were applying the change log guidelines to them or not.

Reply:
They were already mostly following it so I fixed up the version property in their project and added this entry to get them updated and now our tools will help them with version bumps as well as ensuring they have change logs.